src/content/mods.cpp

```diff
@@ -74,7 +77,9 @@ void parseModContents(ModSpec &spec)
 			mod_conf_has_depends = true;
 			std::string dep = info.get("depends");
 			dep.erase(std::remove_if(dep.begin(), dep.end(),
-					static_cast<int(*)(int)>(&std::isspace)), dep.end());
+						  static_cast<int (*)(int)>(
+								  &std::isspace)),
```

Comment:
You may also use one tabulator as indent if visual space is an issue.
Wrong/Mixed indents.

Reply:
This was done by clang-format, please fix that instead